Microsoft.Toolkit.Uwp.SampleApp/SamplePages/PrintHelper/PrintHelperPage.xaml

```diff
@@ -25,54 +47,81 @@
             <Grid x:Name="Container"
                   Grid.RowSpan="2"
                   Opacity="0" />
+            <Grid x:Name="CustomPrintContainer"
+                  Opacity="0" />
             <Grid x:Name="DirectPrintContainer">
-                <StackPanel x:Name="PrintableContent"
-                            Margin="50,20"
-                            Orientation="Vertical">
-                    <Image Width="200"
-                           Source="/Assets/Photos/BigFourSummerHeat.jpg"
-                           Stretch="Fill" />
-                    <TextBlock FontSize="20"
-                               Text="Lorem Ipsum is simply dummy text of the printing and typesetting industry."
-                               TextWrapping="Wrap" />
-                    <TextBlock FontSize="18"
-                               Text="Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book."
-                               TextWrapping="Wrap" />
-                    <TextBlock FontSize="18"
-                               Text="It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged."
-                               TextWrapping="Wrap" />
-                    <TextBlock FontSize="18"
-                               Text="It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum."
-                               TextWrapping="Wrap" />
-                    <TextBlock FontSize="14"
-                               FontStyle="Italic"
-                               Text="From http://www.lipsum.com/"
-                               TextWrapping="Wrap" />
-                </StackPanel>
+                <ListView x:Name="PrintableContent"
```

Comment:
Would it be worth putting the `PrintableContent` as a Grid and then have a [XamlRoot element](https://github.com/Microsoft/UWPCommunityToolkit/blob/master/Microsoft.Toolkit.Uwp.SampleApp/readme.md) in here and put the listview in a .bind file so someone can use the live XAML editor in order to change the content to print?

Reply:
Done.